sdk/notificationhubs/notification-hubs/CHANGELOG.md

```diff
@@ -1,6 +1,13 @@
 # Release History
 
-## 1.0.0 (2023-02-16)
+## 1.0.0-beta.8 (Unreleased)
```

Comment:
beta.9? CHANGELOG shows that beta.8 was out on 2022-11-21

Reply:
Fixed